x-pack/plugins/infra/public/containers/logs/log_position/log_position_state.ts

```diff
@@ -7,9 +7,15 @@
 import { useState, useMemo, useEffect, useCallback } from 'react';
 import createContainer from 'constate';
 import { TimeKey } from '../../../../common/time';
+import { datemathToEpochMillis, isValidDatemath } from '../../../utils/datemath';
 
 type TimeKeyOrNull = TimeKey | null;
 
+interface DateRange {
```

Comment:
We already have a `TimeRange` type in use. Do we want to re-use that?

Reply:
[this one?](https://github.com/elastic/kibana/blob/ece2c35de130aebeba28af4ab9eebe21cd465f15/src/plugins/data/common/timefilter/types.ts#L25-L28). Otherwise there are [a lot of them](https://github.com/search?q=%22interface+TimeRange%22+repo%3Aelastic%2Fkibana&type=Code) :\